README.md

```diff
@@ -63,9 +88,4 @@ where
 then one can compute gradients by providing perturbations
 ```julia
 grads = backward(diff, dA, db, dc)
-```
```

Comment:
We should keep that the project was initially supported by GSOC

Reply:
added